eldritch/events.py

```diff
@@ -4472,6 +4508,51 @@ def animated(self):
     return True
 
 
+class RemoveToken(Event):
```

Comment:
Would it be easier to just also account for zero tokens in the AddToken event, and allow AddToken to accept negative numbers (which it currently can, but we don't floor tokens at zero right now).